torch/_guards.py

```diff
@@ -1,7 +1,9 @@
 import dataclasses
 import enum
 import weakref
-from typing import Callable, List, Optional
+from typing import Callable, List, NamedTuple, Optional
+
+import sympy
```

Comment:
wtf?

Reply:
I think it's because the lint env isn't installing sympy